### PR TITLE
Add latest tests

### PR DESCRIPTION
**Pull request by farshidtz**
Pending PRs:
- https://github.com/canonical/edgex-checkbox-provider/pull/5
- https://github.com/canonical/edgex-checkbox-provider/pull/7

### CODE REVIEW

File: data/latest/summary [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ 
+test-self-refresh-services.sh
+ 
+test-services-alive.sh
+ 
+test-sys-mgmt-agent.sh
+ 
+utils.sh
+
+a=${a:b}
```
Comment (farshidtz) [inline data/latest/summary]: Added by mistake?

Comment (MonicaisHer) [reply data/latest/summary]: Yes, I committed it by mistake.

**Review by MonicaisHer — APPROVED** [review]

File: data/latest/summary (hunk 1 above)
Comment (MonicaisHer) [reply data/latest/summary]: I have deleted it.

**Comment by farshidtz** [comment]
Let's wait for the `jakarta/test-security-services-proxy-certs.sh` fixes and then add them to latest and merge together,

**Comment by farshidtz** [comment]
@MonicaisHer 
Could you please also incorporate the recent changes in [jakarta/test-security-services-proxy-certs.sh](https://github.com/canonical/edgex-checkbox-provider/blob/master/data/jakarta/test-security-services-proxy-certs.sh) into latest tests?

Everything is fixed now.

**Comment by MonicaisHer** [comment]
@farshidtz  I have incorporated and tested it.

**Pull request merged**